### PR TITLE
feat(cd): add ci/cd workflow

### PR DESCRIPTION
**Pull request by schwannden**

### CODE REVIEW

**Comment by lowkingshih** [comment]
Cd player

**Comment by schwannden** [comment]
> Cd player

image

File: src/main.ts [hunk 1]
```diff
@@ -3,6 +3,6 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  await app.listen(3000);
+  await app.listen(8080);
```
Comment (lowkingshih) [inline src/main.ts:6]: Does the 8080 Port have any special uses for the Google Platform? 
What happens if we use  3000 port?

File: .gcloudignore
Comment (lowkingshih) [inline .gcloudignore:1]: Are the` test/ ` folder not uploaded because they won't be run on Google Cloud?

**Comment by schwannden** [comment]
Port 8080 is required for google app engine

lowking shih ***@***.***>於 2023年10月31日 週二，上午10:23寫道：

**Review by lowkingshih — APPROVED** [review]
Amazing workflow. CI/CD is truly an art form, a pleasure to look at.

**Comment by lowkingshih** [comment]
This is my first time doing a code review. 
If there's any room for improvement, I'd appreciate some suggestions.

**Pull request merged**